UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.API.FunctionalTests/FunctionalTests/ExchangeSetForLargeFile.cs

```diff
@@ -31,9 +35,11 @@ public async Task SetupAsync()
             AuthTokenProvider authTokenProvider = new AuthTokenProvider();
             EssJwtToken = await authTokenProvider.GetEssToken();
             FssJwtToken = await authTokenProvider.GetFssToken();
+            ScsApiClient = new SalesCatalogueApiClient(Config.ScsAuthConfig.BaseUrl);
+            ScsJwtToken = await authTokenProvider.GetScsToken();
             DataHelper = new DataHelper();
             ApiEssResponse = await ExchangeSetApiClient.GetProductIdentifiersDataAsync(DataHelper.GetProductIdentifiersForLargeMedia(), accessToken: EssJwtToken);
-            Thread.Sleep(5000); //File creation takes time
+            Thread.Sleep(20000); //File creation takes time as we are using 10 products now.
```

Comment:
Updated. Thanks.

Reply:
this still seems to be 20 sec.

Reply:
Hi Raymond, I have removed it from the cs file(Please find the below screenshot from github).
![image](https://user-images.githubusercontent.com/85937324/184283908-67501ed1-b8b6-418a-be65-0bfcc7d37499.png)


But, we will use this locally to run the FT's, because as discussed with you without sleep it fails locally.